packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -223,6 +225,12 @@ export default {
         if (name === 'useRef' && id.type === 'Identifier') {
           // useRef() return value is stable.
           return true;
+        } else if (
+          (name === 'experimental_useEvent' || name === 'useEvent') &&
+          id.type === 'Identifier'
+        ) {
+          // useEvent() return value is stable.
```

Comment:
Should we add some tests for this?

Reply:
You added [tests for this](https://github.com/facebook/react/blob/f85de771f86436cd082f5dcbd6e6d8a06883b83d/packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js#L1455-L1466), unless you're suggesting more?

Reply:
Oh I forgot :)